bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2415,8 +2419,8 @@ void ReplicaImp::onTransferringCompleteImp(SeqNum newStateCheckpoint) {
   }
 
   if (askAnotherStateTransfer) {
```

Comment:
As you ask for state transfer also when tryToAskForMissingInfo return false because isCollectingState is true, I think we should also add here that if isCollectingState is true, we won't issue state transfer

Reply:
we cannot be in collecting state here